documentation/docs/components/text.md

```diff
@@ -0,0 +1,33 @@
+# Text
+## Example
+
+| type      | name            | value                       |parameter_list |
+| --------- | ------------    | ------                      |--------- |
+|text	    |text_formatting_1|Style parameter Standard     |style: standard|
+|text       |text_formatting_2|Style parameter Contextual	|style: contextual|
+|text       |text_formatting_3|Style parameter Emphasised  	|style: emphasised|
+|text       |text_formatting_4|Style parameter Alternative  |style: alternative|
+
+![](images/text.png)
```

Comment:
Can you make sure that the image and the table agree? (so the image is what the app would show if you created the table as a template)

<img width="299" alt="image" src="https://user-images.githubusercontent.com/74557272/164187820-b75040ad-1f89-44a2-82d5-83de786e2fd6.png">